src/jit/lower.cpp

```diff
@@ -1863,6 +1872,135 @@ GenTree* Lowering::LowerTailCallViaHelper(GenTreeCall* call, GenTree* callTarget
     return result;
 }
 
+void Lowering::LowerCompare(GenTree* cmp)
```

Comment:
Good catch. I agree that the other one should be changed, presumably to `TreeNodeInfoInitCompare` or `TreeNodeInfoInitCmp`

---
In reply to: [78805689](https://github.com/dotnet/coreclr/pull/7189#discussion_r78805689) [](ancestors = 78805689)